src/script/api/script_types.hpp

```diff
@@ -104,11 +104,9 @@
 #ifndef SCRIPT_TYPES_HPP
 #define SCRIPT_TYPES_HPP
 
-#include "../../core/overflowsafe_type.hpp"
-#include "../../company_type.h"
-#include "../../tile_type.h"
 #include <squirrel.h>
 
+#ifdef DOXYGEN_API
```

Comment:
I guess this just means that if we add a type and forget to update this, it only breaks the documentation, nothing else?

Reply:
Yes, exactly that. And I think it's already broken as there are way more IDs that are not exposed in this file/table. For example LeagueTableID and LeagueTableElementID.

Reply:
I've added the 4 missing elements to the table for completeness: ClientID, CompanyID, LeagueTableID and LeagueTableElementID.